index.html

```diff
@@ -40,150 +50,167 @@ <h3 class="subject_text">I'm a Software Developer</h3>
             <li id="M_icon"><a href="#"><img src="Icons\Medium.png" alt="M-symbol link"></a></li>
         </ul>
         <div class="illustration"></div>
+        <div class="desktop_illustration"></div>
+        <div class="desktop_illustration2"></div>
+        <div class="desktop_illustration3"></div>
+        <div class="desktop_illustration4"></div>
+        <div class="desktop_illustration5"></div>
+        <div class="desktop_illustration_line"></div>
+        <div class="desktop_illustration_triangle"></div>
```

Comment:
- [ ] I have noticed that you are using different image backgrounds for an image that should be one. I suggest that you use only on background in the banner. This is to make sure that your portfolio is professional and easy to maintain.
- [ ] In addition, the background image should not have its `div` instead it should be used in the section or under HTML tag. For example for this particular case, the background image should be attached to `headline` ☺️.
Hint: You can consider exporting the illustration header which will download the whole image.
![image](https://github.com/lewisjn-arch/Mobile-Portfolio-setup/assets/99394374/1eba4f0a-b2c8-4ae9-8535-fa0607e0e3e1)